tests/ignite/contrib/metrics/test_precision_recall_curve.py

```diff
@@ -28,7 +28,7 @@ def test_no_sklearn(mock_no_sklearn):
 def test_precision_recall_curve():
     size = 100
     np_y_pred = np.random.rand(size, 1)
-    np_y = np.zeros((size,), dtype=np.long)
+    np_y = np.zeros((size,), dtype=np.int_)
```

Comment:
Can't we do simply `np_y = np.zeros((size,))` without specifying the dtype ? By default it will be equivalent to `long` ?

Reply:
Sure I can update it that way as well.